sentry/src/main/java/io/sentry/Scope.java

```diff
@@ -165,6 +165,7 @@ public void setUser(final @Nullable User user) {
    *
    * @return the fingerprint list
    */
+  @SuppressWarnings("KotlinPropertyAccess")
```

Comment:
they cant be public, users only have access to setters, but it cant be private as well, we need to access from a different package

Reply:
maybe we should add `KotlinPropertyAccess` as excluding rule, otherwise, it's gonna be annoying